integrations/access/servicenow/plugindata.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020-2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicenow
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+// DecodePluginData deserializes a string map to PluginData struct.
+func DecodePluginData(dataMap map[string]string) (data PluginData) {
+	data.User = dataMap["user"]
+	if str := dataMap["roles"]; str != "" {
+		data.Roles = strings.Split(str, ",")
+	}
+	if str := dataMap["created"]; str != "" {
+		var created int64
+		fmt.Sscanf(dataMap["created"], "%d", &created)
```

Comment:
should we check for errors returned by `fmt.Sscanf` or use `strconv.ParseInt`. 
I don't recommend `fmt.Sscanf` because it ignores any spaces existent in the `created` data.
Check out: https://go.dev/play/p/vKD_wgyI7TX